### PR TITLE
4192: Document Type dropdown display during upload#15649

### PR DESCRIPTION
**Pull request by zurbergram**
## Description

As a va.gov developer I need to filter the display of document types during the upload process so that the user is not overwhelmed with the full list of document types and the document types that are displayed to the user are relevant.

## Testing done
- [x] local testing
- [x] unit testing

## Screenshots
screen shot 2019-01-15 at 4 25 26 pm

## Acceptance criteria
- [ ] During the interview if the user specifically selects to **upload a completed** 4192 then the document type list will automatically default to the Form name (_VA 21-4192- Request for Employment Information in Connection with Claim for 
- [x] In the scenario where the Document Type is defaulted to a specific form then it is not editable by the user
- [x]  Document name field will not be editable, it will default to the file name that is uploaded
- [x]  User is able to delete the file if needed
- [x]  User can add another document
- [x]  User can continue to next step in the flow

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/all-claims/config/4192/index.js [hunk 1]
```diff
@@ -2,16 +2,23 @@ import _ from 'lodash';
...
 const isDownloading = formData =>
-  _.get(formData, 'view:upload4192Choice.view:download4192', false);
-// const isUploading = formData => _.get(formData, 'view:upload4192Choice.view:upload4192', false);
+  _.get(formData, 'view:upload4192Choice.view:download4192', false) &&
+  needsToEnterUnemployability(formData);
+
+const isUploading = formData =>
+  _.get(formData, 'view:upload4192Choice.view:upload4192', false) &&
+  needsToEnterUnemployability(formData);
+
```
Comment (imeecuison) [inline src/applications/disability-benefits/all-claims/config/4192/index.js]: Should these be in utils?

Comment (zurbergram) [reply src/applications/disability-benefits/all-claims/config/4192/index.js]: figured with the eventual logic that will have to go into the index made more sense to put them here, also they are only going to be used here

**Review by imeecuison — APPROVED** [review]
Just one comment. You can leave it there. Doesn't break anything. LGTM

**Comment by zurbergram** [comment]
@department-of-veterans-affairs/code-reviewers-for-external-teams-group - Could I get a code review on this at your earliest convenience? Thanks!

File: src/applications/disability-benefits/all-claims/content/pastEmploymentFormDownload.jsx [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-import { VA_FORM4142_URL } from '../constants';
+import { VA_FORM4192_URL } from '../constants';
 import { claimsIntakeAddress } from './itfWrapper';
 
 export const download4192Notice = (
   <div>
     <p>
-      <a href={VA_FORM4142_URL} target="_blank">
-        Download VA Form 21-4142
+      <a href={VA_FORM4192_URL} target="_blank">
```
Comment (jbalboni) [inline src/applications/disability-benefits/all-claims/content/pastEmploymentFormDownload.jsx]: Can you add `rel="noopener"` to this? It's a good practice for links opening in a new tab.

File: src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.js [hunk 3]
```diff
@@ -39,7 +39,10 @@ export const uiSchema = {
       },
       items: {
         name: {
-          'ui:title': 'Name of hospital',
+          'ui:title': 'Hospital’s name',
+        },
+        'view:hospitalAddressTitle': {
+          'ui:title': "Hospital's address",
```
Comment (jbalboni) [inline src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.js]: Should use an `’` here, too.

**Review by jbalboni — APPROVED** [review]
Couple minor things, but LGTM otherwise

**Pull request merged**